wls-auth-service/src/test/java/de/muenchen/oss/wahllokalsystem/authservice/domain/LoginAttemptRepositoryTest.java

```diff
@@ -2,6 +2,7 @@
 
 import de.muenchen.oss.wahllokalsystem.authservice.MicroServiceApplication;
 import de.muenchen.oss.wahllokalsystem.authservice.TestConstants;
+import de.muenchen.oss.wahllokalsystem.authservice.configuration.Profiles;
```

Comment:
macht es einen Unterschied, ob man die Klasse importiert, oder über einen static import direkt die Konstante? Wir haben es im Projekt mal so, mal so.

Reply:
Spielt für die Ausführung keine Rolle. Es verkürzt lediglich den Code da man sich die Klasse sparen kann, was den Code in machen Fällen lesbarer machen kann.

Habe im Rahmen andere Issues und PRs auch gesehen dass es unterschiedlich ist. Aus meiner Sicht ist dies aktuell kein relevantes Thema.